include/pybind11/pybind11.h

```diff
@@ -1336,6 +1336,11 @@ class module_ : public object {
         if (doc && options::show_user_defined_docstrings()) {
             result.attr("__doc__") = pybind11::str(doc);
         }
+
+        // GraalPy doesn't support PyModule_GetFilenameObject,
```

Comment:
This is fine if you want to keep it, but the official `PyModule_GetFilenameObject()` seems most future proof, compared to referencing `"__file__"` directly.

We have 7 `defined(GRAALVM_PYTHON)` already under include/pybind11/, I'd lean towards adding another one here.

Reply:
Okay, done.